doc/api/process.md

```diff
@@ -1651,7 +1651,9 @@ added: v0.2.0
 * {Object}
 
 The `process.versions` property returns an object listing the version strings of
-Node.js and its dependencies.
+Node.js and its dependencies. In addition, `process.versions.modules` indicates
+the current ABI version, which is increased whenever a C++ API changes. Node.js
+will refuse to load native modules built for an older `modules` value.
```

Comment:
nit: This sounds like Node will load native modules built for *newer* `modules` values, which isn’t the case… maybe something like `for a different` fits better?